examples/malfunction-indicator-light/mail-notification/pushImage.sh

```diff
@@ -0,0 +1,54 @@
+ #!/bin/bash
+######################################################################
+# Copyright (c) 2019 Bosch Software Innovations GmbH [and others]
+#
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+########################################################################
+
+# Make the script fail if a command fails
+set -e
+
+SCRIPTPATH=$(dirname "$(readlink -f "$0")")
+
+DOCKER_REGISTRY_SERVER=$1
+DOCKER_REGISTRY_USERNAME=$2
+DOCKER_REGISTRY_PASSWORD=$3
+
+DOCKER_IMAGE_NAME="mail-notification"
+VERSION="$(gradle properties -q | grep 'version:' | awk '{print $2}')"
```

Comment:
Was it you intention to read the version from gradle in this pushImage.sh instead of setting it from a command-line parameter? A previous PR replaced that behaviour by setting VERSION explicitly in another pushImage.sh.

I'm fine with both approaches for now, just stumbled over the difference.

Reply:
It was not my intention and I will adapt the version handling in the PR.